synapsis/src/main/resources/templates/pages/createJob.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
```

Comment:
Why does this html file have a header? Does it not exist as a fragment within the layout?